openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
Yes and no. If I want to project the same motion on two projectors (which should be possible by server side) I also get different uuids. But I do not want to save uuids to every model we have. So we should use only one way I think.